README.md

```diff
@@ -965,112 +1408,535 @@ export function trap(message: string): Query<boolean> {
+| stable read int16    | :x:                                                                                          | :heavy_check_mark:                                                                                                       | :x:                |
+| stable write int16   | :x:                                                                                          | :heavy_check_mark:                                                                                                       | :x:                |
+| stable read int8     | :x:                                                                                          | :heavy_check_mark:                                                                                                       | :x:                |
+| stable write int8    | :x:                                                                                          | :heavy_check_mark:                                                                                                       | :x:                |
+| stable read float64  | :x:                                                                                          | :heavy_check_mark:                                                                                                       | :x:                |
+| stable write float64 | :x:                                                                                          | :heavy_check_mark:                                                                                                       | :x:                |
 
 ### Roadmap
 
--   [ ] 1.0
-    -   [ ] [Feature parity with Rust and Motoko CDKs](https://github.com/demergent-labs/azle/issues/134)
-    -   [ ] Core set of Azle-specific npm packages
-    -   [ ] Sudograph integration
-    -   [ ] Official dfx integration with `"type": "typescript"` or `"type": "azle"`
-    -   [ ] Live robust examples
-    -   [ ] Video series
-    -   [ ] Comprehensive benchmarks
-    -   [ ] Robust property-based tests
-    -   [ ] Optimized compilation
-    -   [ ] Security audits
--   [ ] 2.0
-    -   [ ] Azle VS Code plugin
-    -   [ ] [Inter-Canister Query Calls](https://forum.dfinity.org/t/inter-canister-query-calls-community-consideration/6754)
-
-### Limitations
+-   [ ] July 2022
+    -   [ ] Extensive automated benchmarking
+-   [ ] August 2022
+    -   [ ] Compiler error DX revamp
+    -   [ ] Rust rewrite
+-   [ ] September 2022
+    -   [ ] Extensive automated property testing
+    -   [ ] Multiple independent security reviews/audits
 
+### Gotchas and Caveats
+
+-   Because Azle is built on Rust, to ensure the best compatibility use underscores to separate words in directory, file, and canister names
```

Comment:
And maybe add methods. It might be implied but explicitly telling people to name their canister methods in underscores would be good.